plugin/kubernetes/controller.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"github.com/coredns/coredns/plugin/kubernetes/object"
+	"github.com/coredns/coredns/plugin/metrics/vars"
```

Comment:
there is no reason to include this, you can just define a bunch of metrics in the plugin and export those.

Reply:
Done.